spring-integration-sftp/src/test/java/org/springframework/integration/sftp/inbound/RollbackLocalFilterTests-context.xml

```diff
@@ -31,7 +30,7 @@
 	<bean id="acceptOnceFilter" class="org.springframework.integration.file.filters.AcceptOnceFileListFilter" />
 
 	<int:transaction-synchronization-factory id="syncFactory">
-		<int:after-rollback expression="@acceptOnceFilter.remove(payload)" />
+		<int:after-rollback expression="payload.delete()" />
```

Comment:
The remote persistence file list filter is something new from the `master` now.
Didn't think about that yet 😄 

All other your concerns to this test unrelated.
That was a behavior to remove from the filter, now I want to check if file will come again with the watchService and when I delete it.

Since it isn't stable on all platforms we shouldn't rely on the `WatchService` by default.
Therefore this test must be reinstated with its original purpose. 
That's it.